spec/index.html

```diff
@@ -295,16 +296,20 @@ <h3>RDF Vocabularies and Namespace IRIs</h3>
       </tr>
     </table>
 
-    <p>In some serialization formats it is common to abbreviate <a>IRIs</a>
-      that start with <a>namespace IRIs</a> by using a
-      <a>namespace prefix</a> in order to assist readability. For example, the IRI
-      <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral</code>
+    <p>In some serialization formats, it is common
+      to associate some [=namespace IRIs=] with arbitrary [=namespace prefixes=],
+      and to improve readability by abbreviating <a>IRIs</a> that start with one of those <a>namespace IRIs</a>
+      by using the corresponding <a>namespace prefix</a>.
+      For example,
+      based on the prefix mapping in <a href="#tab-vocab-ns">the table above</a>,
+      the IRI <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral</code>
       would be abbreviated as <code>rdf:XMLLiteral</code>.
-      Note however that these abbreviations are <em>not</em> valid IRIs,
-      and must not be used in contexts where IRIs are expected.
-      Namespace IRIs and namespace prefixes are <em>not</em> a formal part of the
-      RDF data model. They are merely a syntactic convenience for
-      abbreviating IRIs.</p>
+      Note however that such abbreviations are <em>not</em> meant to be processed directly as IRIs,
+      and must not be used in syntactic contexts where IRIs are expected.
```

Comment:
I've lost track of normativity. Should this use `MUST` instead of `must`?
```suggestion
      and must not be used in syntactic contexts where IRIs are expected.
```

Reply:
No. This is in §1.4; §1 (Introduction) as a whole is informative.

Reply:
Does this work?
```suggestion
      and are not to be used in syntactic contexts where IRIs are expected.
```